### PR TITLE
Make scrabble less scrabbly

### PR DESCRIPTION
**Pull request by philmcmahon**
Some people have never played scrabble before. This change attemps to make this challenge easier to follow if you haven't played scrabble.

### CODE REVIEW

File: scrabble/README.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
-Scrabble
---------
+Word Game (Scrabble)
+--------------------
+![scrabble](https://www.word-grabber.com/3/m/2012/02/scrabble-rack-with-tiles.jpg)
+
+This is a game where players attempt to create words from a set of letter tiles.  Different letters have different scores
```
Comment (jeteve) [inline scrabble/README.md]: Maybe 'different point values'

File: scrabble/README.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+
+* Calculate the score for a word. Use these scores for each letter:
```
Comment (jeteve) [inline scrabble/README.md]: "defined as the sum of its letter point values"?

Comment (akash1810) [reply scrabble/README.md]: Should we give an example?

```
GUARDIAN = 2 + 1 + 1 + 1 + 2 + 1 + 1 + 1 = 10
```

File: scrabble/README.md [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
+    - 1 point: E, A, I, O, N, R, T, L, S, U 
+    - 2 points: D, G 
+    - 3 points: B, C, M, P 
+    - 4 points: F, H, V, W, Y 
+    - 5 points: K 
+    - 8 points: J, X 
+    - 10 points: Q, Z 
+
+* Assign seven tiles chosen randomly from the english alphabet to a player's rack.
+* In this game, letters are taken at random from a 'bag' containing a fixed number of each letter. Assign seven tiles to a rack using a bag with this distribution:
```
Comment (paulbrown1982) [inline scrabble/README.md]: 'In the real game...'

**Pull request merged**